src/actor/Actor.ts

```diff
@@ -12,7 +12,7 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-import { WatchableMessageStore } from "../watchable-message-store/WatchableMessageStore.js";
+import { ActorRealm } from "./ActorRealm.js";
 
 declare global {
```

Comment:
No need to address it in this PR, but we probably need to keep in mind that with multiple (unconnected) realms it’s technically okay to have multiple actors with the same name but different messaging contracts.